packages/lesswrong/server/scripts/exportPostDetails.ts

```diff
@@ -66,10 +71,10 @@ Vulcan.exportPostDetails = wrapVulcanAsyncScript(
     const rows: Array<any> = []
     for (let post of documents.fetch()) {
       // SD: this makes things horribly slow, but no idea how to do a more efficient join query in Mongo
```

Comment:
Aw, nice that this code is getting a little workout again :)

Reply:
:)